tests/tensor/rewriting/test_math.py

```diff
@@ -4571,3 +4571,22 @@ def test_log_kv_stabilization():
         out.eval({x: 1000.0}, mode=mode),
         -1003.2180912984705,
     )
+
+
+@pytest.mark.parametrize("shape", [(), (4, 5, 6)], ids=["scalar", "tensor"])
+def test_pow_1_rewrite(shape):
+    x = pt.tensor("x", shape=shape)
+    z = 1**x
+
+    f1 = pytensor.function([x], z, mode=get_default_mode().excluding("canonicalize"))
+    assert debugprint(f1, file="str").count("Pow") == 1
```

Comment:
Please don't do tests will debugprint. Last time I tweaked string representation had to change a bazillion tests.

f1.maker.fgraph.apply_nodes and check how many are instance of pow.

Reply:
Yeah it seemed bad, but I copied this from another test. I couldn't figure out how to check for pow because it's an Elemwise and not a type (`isinstance(x, pt.pow)` errors, for example)

Reply:
pt.pow is an instance not a type. You can check equality or check is isntance Elemwise and scalar op is Instance Pow